src/main/resources/services/getMarkdownNodes/getMarkdownNodes.ts

```diff
@@ -0,0 +1,30 @@
+import { connectMarkdownRepo } from '/lib/ssb/utils/markdownUtils'
+
+export const get = (): XP.Response => {
```

Comment:
This service could be probably be rewritten slightly and moved to `/utils/` instead of a service since its purpose is solely fetching data from the repository, and is called within an existing service already.